src/debug/inc/arm/primitives.h

```diff
@@ -30,7 +30,11 @@ typedef DPTR(CORDB_ADDRESS_TYPE)    PTR_CORDB_ADDRESS_TYPE;
 #define STACKWALK_CONTROLPC_ADJUST_OFFSET 2
 
 #define CORDbg_BREAK_INSTRUCTION_SIZE 2
+#ifdef LINUX32
```

Comment:
Why would they be Linux specific? I assume that all Unixes that support hardware signals would likely handle these in the same way. 
Also, we don't have any usage of LINUX32 symbol in our codebase, so I am hesitant to introduce it. In places where we need Linux specific features, we use `__linux__` ifdef.